ansible/group_vars/riakcs.yml

```diff
@@ -0,0 +1,7 @@
+#datavol_device=... # defined in commcare-hq/fab/inventories/* if applicable
+# http://docs.basho.com/riak/latest/ops/tuning/linux/#Storage-and-File-System-Tuning
+datavol_opts: barrier=0,data=writeback
```

Comment:
no such directory or file exists. `roles/riakcs` contains multiple sub-roles. Also, it's referenced in `roles/ecryptfs/tasks/main.yml`, so I'm not sure if putting it in the `riakcs` role would work.